testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksColumnMappingMode.java

```diff
@@ -0,0 +1,176 @@
+                    .containsOnly(expectedRows);
+            assertThat(onTrino().executeQuery("SELECT a_number, array_col[1].array_struct_element, nested.field1, a_string FROM delta.default." + tableName))
+                    .containsOnly(expectedRows);
+            assertThat(onTrino().executeQuery("SELECT a_string FROM delta.default." + tableName + " WHERE a_number = 1"))
+                    .containsOnly(ImmutableList.of(row("ala")));
+            assertThat(onTrino().executeQuery("SELECT a_number FROM delta.default." + tableName + " WHERE nested.field1 = 'databricks 1'"))
+                    .containsOnly(ImmutableList.of(row(1)));
+
+            // Verify the connector can read renamed columns correctly
+            onDelta().executeQuery("ALTER TABLE default." + tableName + " RENAME COLUMN a_number TO new_a_column");
+            onDelta().executeQuery("ALTER TABLE default." + tableName + " RENAME COLUMN nested.field1 TO field2");
+
+            assertThat(onTrino().executeQuery("DESCRIBE delta.default." + tableName))
+                    .containsOnly(ImmutableList.of(
+                            row("new_a_column", "integer", "", ""),
+                            row("array_col", "array(row(array_struct_element varchar))", "", ""),
+                            row("nested", "row(field2 varchar)", "", ""),
+                            row("a_string", "varchar", "", "")));
+
+            assertThat(onDelta().executeQuery("SELECT new_a_column, array_col[0].array_struct_element, nested.field2, a_string FROM default." + tableName))
+                    .containsOnly(expectedRows);
+            assertThat(onTrino().executeQuery("SELECT new_a_column, array_col[1].array_struct_element, nested.field2, a_string FROM delta.default." + tableName))
+                    .containsOnly(expectedRows);
+        }
+        finally {
+            onTrino().executeQuery("DROP TABLE delta.default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testShowStatsForColumnMappingModeNameTable()
+    {
+        String tableName = "test_dl_show_stats_for_column_mapping_mode_" + randomTableSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                " (a_number INT)" +
+                " USING delta " +
+                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                " TBLPROPERTIES (" +
+                " 'delta.columnMapping.mode' = 'name'," +
+                " 'delta.minReaderVersion' = '2'," +
+                " 'delta.minWriterVersion' = '5')");
+
+        try {
+            onDelta().executeQuery("INSERT INTO default." + tableName + " VALUES (1), (2), (null)");
+
+            assertThat(onTrino().executeQuery("SHOW STATS FOR delta.default." + tableName))
+                    .containsOnly(ImmutableList.of(
```

Comment:
According to the documentation. https://trino.io/docs/current/sql/show-stats.html `row_count` is supposed to be `NULL` in the column statistics rows.
Is the documentation false or is the output of the query false?

Reply:
`row_count` column position is at 5, so the value is null in this test. Or am I misunderstanding your question? 
```
row("a_number", null, null, 0.33333333333, **null**, "1", "2"),
```

Reply:
I was actually referring to the summary row.

> `row_count` NULL in column statistic rows

```
row(null, null, null, null, 3.0, null, null)
```

Reply:
"row_count NULL in column statistic rows" means all rows except for bottom are always null in my understanding.